.circleci/config.yml

```diff
@@ -393,6 +409,7 @@ workflows:
   version: 2
   build_test_deploy:
     jobs:
+      - pypi_precheck
```

Comment:
is this running on every build or just tagged ones?

Reply:
Every build. The idea is to get an error before we tag and have to re-tag. I was only doing it on `rel/*` branches, but it's a 2min job.

Reply:
I'll move it back in the rel branch.